sdk/src/feature_set.rs

```diff
@@ -665,6 +665,10 @@ pub mod checked_arithmetic_in_fee_validation {
     solana_sdk::declare_id!("5Pecy6ie6XGm22pc9d4P9W5c31BugcFBuy6hsP2zkETv");
 }
 
+pub mod last_restart_slot_sysvar {
+    solana_sdk::declare_id!("C4QcP9KXjMLirtZGAW4AFzqLH99PaHjsxYx1N48V5LF");
```

Comment:
Just posting this comment as a reminder: we'll prefer to swap out this keypair for one held by a Solana Labs employee before landing.

Reply:
Yes i had already guessed that. I can recommend few keypairs of my own :)

Reply:
```suggestion
    solana_sdk::declare_id!("HooKD5NC9QNxk25QuzCssB8ecrEzGt6eXEPBUxWp1LaR");
```
This is a keypair that I hold. I totally didn't grind that prefix!

Reply:
```suggestion
    solana_sdk::declare_id!("HooKD5NC9QNxk25QuzCssB8ecrEzGt6eXEPBUxWp1LaR");
```
This is a keypair that I hold. I totally didn't grind that prefix!